electron/api/recordJourney.ts

```diff
@@ -25,7 +25,7 @@ import path from 'path';
 import { EventEmitter, once } from 'events';
 import { existsSync } from 'fs';
 import { BrowserWindow, ipcMain, IpcMainInvokeEvent } from 'electron';
-import type { BrowserContext } from 'playwright-core';
```

Comment:
@kyungeunni only changing it because the linter complained the types didn't match between this and the interface of `openPage` further down in this file.